File: help/eng_gb/exercicios_da_ficha_9/Ficha09_011/Ficha09_011.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_011 {

    //Main Function
    static void Main() {
        Console.Write("Number : " );
        int num = int.Parse(Console.ReadLine());
        Console.Write("Bits = " );
        while(num > 0 ){
            Console.Write(num % 2 + " " );
            num /=2;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_011/Ficha09_011.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha09_011 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Number : " );
        int num = Integer.valueOf(keyb.nextLine());
        System.out.print("Bits = " );
        while(num > 0 ){
            System.out.print(num % 2 + " " );
            num /=2;
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12
